### DISCUSSION TITLE
Export DB TAble - Update or Insert

### DISCUSSION THREAD

**Discussion by ObjectAssign** [dbeaver/dbeaver#18674] [Q&A]
When exporting database tables, i dont see the option to make an update or insert sql? its all Insert?

Is this possible?

**Comment by ObjectAssign** [comment]
I found it never mind :) does however produce only sql queries which cannot be ran... so many errors from the SQL it produce...

**Reply by LonwoLonwo** [reply]
Which kind of errors @ObjectAssign ?
